src/autoPagination.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const makeRequest = require('./makeRequest');
-const utils = require('./utils');
+const utils_1 = require('./utils');
```

Comment:
why do we need the rename?

Reply:
I renamed this to address the error `Cannot redeclare block-scoped variable 'utils'.ts(2451)` (due to `utils` already being declared in stripe.js).